tests/unit/formatters/format-relative-test.ts

```diff
@@ -1,4 +1,4 @@
-import { createIntl, OnErrorFn } from '@formatjs/intl';
+import { createIntl, type OnErrorFn } from '@formatjs/intl';
```

Comment:
fixing the deprecated config option brought this up - it's only used as a type, so we should import it as such.